tests/sspi/client_server/kerberos/kdc.rs

```diff
@@ -32,7 +33,7 @@ use rand::{Rng, RngCore};
 use sspi::kerberos::KERBEROS_VERSION;
 use time::{Duration, OffsetDateTime};
 
-pub const MAX_TIME_SKEW: Duration = Duration::minutes(3);
+pub const MAX_TIME_SKEW: StdDuration = StdDuration::from_secs(3);
```

Comment:
suggestion: You may use either time::Duration or std::time::Duration, but there is no need to mix them both.

Reply:
> but there is no need to mix them both.

I don't want to force users to have the `time` crate installed, so I decided to use `std::time::Duration` instead. It is used only in `ServerProperties::max_time_skew` (because it is a part of the public API). It is also convenient, because `time::Duration` implements addition/comparison with `std::time::Duration`

Reply:
> I prefer `std::time::Duration` over renaming types, unless there is really no other better alternative.

done (I accidentally refactored it in my last commits 🙃)